modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/PriceTag.java

```diff
@@ -91,6 +95,22 @@ public int doStartTag() {
 
 			putValue("prices", priceModel);
 
+			CommerceChannel commerceChannel =
+				_commerceChannelService.fetchCommerceChannel(
+					commerceContext.getCommerceChannelId());
+
+			boolean netPrice = true;
```

Comment:
We can sort like this since commerceChannel is only being used to modify netPrice.

```suggestion
			boolean netPrice = true;

			CommerceChannel commerceChannel =
				_commerceChannelService.fetchCommerceChannel(
					commerceContext.getCommerceChannelId());
```